### PR TITLE
[SW-1078] Rename aepp-sdk instances to aeSdk standard

### PR DESCRIPTION
**Pull request by subhod-i**
as we import multiple SDKs for multichain architecture, Naming sdk as aeSdk( as per SDK repo), and other public examples is good.

Removed the `ISDK` type too.

### CODE REVIEW

**Comment by pstachel** [comment]
Shouldn't we also consider renaming the base file for sdk from 'sdk.ts' to 'aeSdk.ts'?

**Comment by subhod-i** [comment]
> Shouldn't we also consider renaming the base file for sdk from 'sdk.ts' to 'aeSdk.ts'?

Of course. I missed it.

**Review by pstachel — APPROVED** [review]

File: src/lib/shSdkWallet.ts [hunk 1]
```diff
@@ -20,7 +20,11 @@ type ISpendOptions = Omit<Parameters<typeof spend>[2], 'onAccount' | 'onNode'>
     payload?: Encoded.Any,
   }
 
-export class ShSdkWallet extends AeSdkWallet {
+/**
+ * Class extends `AeSdkWallet` from aepp-sdk-js
+ * provides flexibility to manage the accounts the way wallet would like to handle
+ */
+export class ShAeSdk extends AeSdkWallet {
```
Comment (peronczyk) [inline src/lib/shSdkWallet.ts:27]: 1. I suggest to use `AeShSdkWallet` or `ShAeSdkWallet` here to maintain the base class naming strategy.
2. Good practice is to name the file the same way as the exported class name.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/lib/shSdkWallet.ts (hunk 1 above)
Comment (subhod-i) [reply src/lib/shSdkWallet.ts]: Renamed it to `AeSdkSupehero` to maintain the base class naming strategy.

AeSdkBase -> AeSdkWallet -> AeSdkSuperhero

**Review by konradodo — APPROVED** [review]

**Review by peronczyk — APPROVED** [review]

**Pull request merged**